nuget.config

```diff
@@ -6,5 +6,6 @@
   <packageSources>
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="vs-devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
```

Comment:
The myget feed is gone. You should be able to remove this.

Reply:
Remove.

Reply:
This was fixed in #234